lib/instrument/transformer/dtx-encode.js

```diff
@@ -21,7 +21,7 @@ class DTXEncoder extends Stream.Transform {
       selector,
       expectsReply: sync
     });
-    if (auxiliaries instanceof DTXMessageAuxBuffer) {
+    if (auxiliaries instanceof DTXMessageAux) {
```

Comment:
it would be nice if we could make this module typed
cc @boneskull

Reply:
Do I have to solve this?

Reply:
you could do it in the future if you have time. It's not for this PR